### PR TITLE
Apply feedback on Convene's landing page

### PR DESCRIPTION
**Pull request by zspencer**
Per, @maximegalon5, I've removed references to client ownership (since we're not really ready to do that yet)

Per @colombene  I've updated the title to be a bit more discoverable and to try and include other parts of our core differentiator(s) from the positioning statement work she did.

I've been using https://docs.google.com/document/d/164RepjCMVhVmk--bvAXvRaDVxvLAloZE8wYMAmUv0e0/edit for the "starting point" for copy; and am happy to apply changes to this page if folks aren't comfortable doing so themselves, or figure out a way to pull in content so we don't have to use github to manage the application of copy-changes. 

Screenshot_2020-05-22 Convene

### CODE REVIEW

File: src/convene.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: layouts/product-landing-page.njk
 title: Convene
-subtitle: Bringing Workers Together
-summaryText: Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui lorem cupidatat commodo. Elit sunt amet fugiat veniam occaecat fugiat aliqua.
+subtitle: &mdash; Video-based Online Collaboration Tools
```
Comment (user512) [inline src/convene.html:4]: Is `&mdash` a unicode for a dash? I am just curious on why isn't it showing on the screenshot.

**Review by user512 — APPROVED** [review]

File: src/convene.html (hunk 1 above)
Comment (zspencer) [reply src/convene.html:4]: Ah, I think it's getting stripped out. It's a unicode symbol for an [em-dash](https://www.compart.com/en/unicode/U+2014)

**Comment by maximegalon5** [comment]
Looks good to me! I like the 4 descriptive points <- revolves around our core Ho Position.

**Review by maximegalon5 — COMMENTED** [review]
Looks good. I like the descriptors which address our Ho Position

**Pull request merged**